Remotion/Web/UnitTests/Core/Utilities/HtmlUtilityTest.cs

```diff
@@ -23,36 +23,6 @@ namespace Remotion.Web.UnitTests.Core.Utilities
   [TestFixture]
   public class HtmlUtilityTest
   {
-    [Test]
-    public void HtmlEncode_DoubleQuote ()
-    {
-      Assert.That (HtmlUtility.HtmlEncode ("a\"b"), Is.EqualTo ("a&quot;b"));
-    }
-
-    [Test]
-    public void HtmlEncode_SingleQuote ()
-    {
-      Assert.That (HtmlUtility.HtmlEncode ("a'b"), Is.EqualTo ("a&#39;b"));
-    }
-
-    [Test]
-    public void HtmlEncode_NewLine ()
-    {
-      Assert.That (HtmlUtility.HtmlEncode ("a\nb"), Is.EqualTo ("a<br />b"));
-    }
-
-    [Test]
-    public void HtmlEncode_LineFeed ()
-    {
-      Assert.That (HtmlUtility.HtmlEncode ("a\rb"), Is.EqualTo ("a<br />b"));
-    }
-
-    [Test]
-    public void HtmlEncode_LineFeedNewLine ()
-    {
-      Assert.That (HtmlUtility.HtmlEncode ("a\r\nb"), Is.EqualTo ("a<br />b"));
-    }
-
```

Comment:
why deleted?

Reply:
Since we are covering those cases in the `WebStringTest` and I thought it would be better to remove them now than to remove them when we remove `HtmlUtility`.

Reply:
Nah, I'd leave them. They don't hurt us and they must never become red anyway or we'd have a problem.